dmscripts/notify_suppliers_with_incomplete_applications.py

```diff
@@ -0,0 +1,59 @@
+from dmapiclient import DataAPIClient
+from dmscripts.helpers import logging_helpers
+from dmscripts.helpers.auth_helpers import get_auth_token
+from dmscripts.helpers.email_helpers import scripts_notify_client
+from dmscripts.helpers.logging_helpers import logging
+from dmscripts.helpers.supplier_data_helpers import AppliedToFrameworkSupplierContextForNotify
+from dmutils.email.exceptions import EmailError
+from dmutils.email.helpers import hash_string
+from dmutils.env_helpers import get_api_endpoint_from_stage
+
+
+def notify_suppliers_with_incomplete_applications(doc_opt_arguments):
+    dry_run = doc_opt_arguments['--dry-run']
+    framework = doc_opt_arguments['<framework>']
+    stage = doc_opt_arguments['<stage>']
```

Comment:
We usually don't let details about the docopt information leak into the `dmscripts` modules. I think one of the reasons for the existence of the separate modules in the  `scripts/` dir is that _they_ are in charge of turning command line parameters into standard python args.